schemas/common_definitions.json

```diff
@@ -297,5 +297,15 @@
       "additionalProperties": false,
       "required": ["goto"]
     }
+  },
+  "submission": {
+    "type": "object",
+    "properties": {
+      "button": {
+        "type": "string",
+        "description": "The text that appears on the submit button, when the questionnaire is ready to be submitted"
+      }
+    },
+    "additionalProperties": false
```

Comment:
This definition is specific to the hub so should be in `schemas/hub/definitions.json`.